### PR TITLE
Add dedicated page for movies

### PR DESCRIPTION
**Pull request by tauhrick**
* Adds dedicated page for movies
Screenshot 2020-08-24 at 10 06 42 PM

* Adds button to home page for showing more details which links to this dedicated page.
Screenshot 2020-08-24 at 10 15 22 PM

### CODE REVIEW

**Review by PratibhaShrivastav — APPROVED** [review]

File: src/main/webapp/WEB-INF/tv_show_dedicated_page.jsp [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                            </button>
+                        </form>
+                        <li class="icon-bar">
+                            <div class="g-signin2" data-onsuccess="onSignIn" data-width="120" data-height="45"></div>
+                        </li>
+                        <li class="icon-bar">
+                            <div class="sign-out">
+                                <button id="signout-button" onclick="signOut()">Sign Out</button>
+                            </div>
+                        </li>
```
Comment (digu-007) [inline src/main/webapp/WEB-INF/tv_show_dedicated_page.jsp:79]: If you are using the complete nav bar, add the latest change for user details as well.

File: src/main/webapp/assets/javascript/book_dedicated_page.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  form.append(field1);
+  var field2 = $('<input></input>');
+  field2.attr('type', 'hidden');
+  field2.attr('name', 'EntityID');
+  field2.attr('value', entityID);
+  form.append(field2);
+  console.log(form);
+  $(document.body).append(form);
+  form.submit();
+}
```
Comment (digu-007) [inline src/main/webapp/assets/javascript/book_dedicated_page.js]: Use AJAX post call.

File: src/main/webapp/assets/javascript/movie_dedicated_page.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  form.append(field1);
+  var field2 = $('<input></input>');
+  field2.attr('type', 'hidden');
+  field2.attr('name', 'EntityID');
+  field2.attr('value', entityID);
+  form.append(field2);
+  console.log(form);
+  $(document.body).append(form);
+  form.submit();
+}
```
Comment (digu-007) [inline src/main/webapp/assets/javascript/movie_dedicated_page.js]: Same

File: src/main/webapp/assets/javascript/tv_show_dedicated_page.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  form.append(field1);
+  var field2 = $('<input></input>');
+  field2.attr('type', 'hidden');
+  field2.attr('name', 'EntityID');
+  field2.attr('value', entityID);
+  form.append(field2);
+  console.log(form);
+  $(document.body).append(form);
+  form.submit();
+}
```
Comment (digu-007) [inline src/main/webapp/assets/javascript/tv_show_dedicated_page.js]: Same

**Pull request closed without merging**